lib/global-eval.js

```diff
@@ -62,6 +64,10 @@ var __exec;
         e = addToError(_e, 'Evaluating ' + load.address);
       }
       preExec(this);
+
+      for (var property in load.metadata)
+        script[property] = load.metadata[property];
```

Comment:
Let's rather set the allowed properties individually. There are object metadatas etc that shouldn't propagate here.